### PR TITLE
Setup client and server

### PR DESCRIPTION
**Pull request by IvanPletinskiy**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
```
Comment (IvanPletinskiy) [inline frontend/package.json:1]: Here dependencies for frontend

File: frontend/src/App.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
```
Comment (IvanPletinskiy) [inline frontend/src/App.js:1]: React widget

File: frontend/src/App.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import logo from "./logo.svg";
+import "./App.css";
+
+function App() {
+  const [data, setData] = React.useState(null);
+
+  React.useEffect(() => {
+    fetch("/api")
```
Comment (IvanPletinskiy) [inline frontend/src/App.js:9]: Make request to localhost:3001/api and wait for result

File: server/index.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+// server/index.js
```
Comment (IvanPletinskiy) [inline server/index.js:1]: Server configuration

File: server/index.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+const PORT = process.env.PORT || 3001;
+
+const app = express();
+
+app.get("/api", (req, res) => {
+    res.json({ message: "Hello from server!" });
+});
+
+app.listen(PORT, () => {
```
Comment (IvanPletinskiy) [inline server/index.js:13]: Listen to port

File: server/index.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+// server/index.js
+
+const express = require("express");
+
+const PORT = process.env.PORT || 3001;
+
+const app = express();
+
+app.get("/api", (req, res) => {
```
Comment (IvanPletinskiy) [inline server/index.js:9]: Register route /api with GET method

**Comment by dawist-o** [comment]
Good job, boy

**Comment by dawist-o** [comment]
@Weizhau technically approved

**Pull request merged**